FVCalc.java

```diff
@@ -1,6 +1,12 @@
 // Computes the future value of a saving investment.
 public class FVCalc {
 	public static void main(String[] args){
-		// Replace this comment with your code
+		int currentValue=Integer.parseInt(args[0]);
+		double rate=(Double.parseDouble(args[1]))/100;
+		int n=Integer.parseInt(args[2]);
+		double future=(currentValue*Math.pow((1+rate), ((double)n)));
```

Comment:
Same here with n, if you want n to be double parse it as a double from the beginning.